pages/blog/asyncapi-and-apicurio-for-asynchronous-apis.md

```diff
@@ -0,0 +1,223 @@
+        url: https://www.apache.org/licenses/LICENSE-2.0
+
+    servers:
+      test:
+        url: test.mykafkacluster.org:8092
+        protocol: kafka-secure
+        description: Test broker
+        security:
+          - saslScram: []
+
+    defaultContentType: application/json
+
+    channels:
+      smartylighting.streetlights.1.0.event.{streetlightId}.lighting.measured:
+        description: The topic on which measured values may be produced and consumed.
+        parameters:
+          streetlightId:
+            $ref: '#/components/parameters/streetlightId'
+        publish:
+          summary: Inform about environmental lighting conditions of a particular streetlight.
+          operationId: receiveLightMeasurement
+          message:
+            $ref: '#/components/messages/lightMeasured'
+
+    components:
+      messages:
+        lightMeasured:
+          name: lightMeasured
+          title: Light measured
+          summary: Inform about environmental lighting conditions of a particular streetlight.
+          contentType: application/json
+          payload:
+            $ref: "#/components/schemas/lightMeasuredPayload"
+
+      schemas:
+        lightMeasuredPayload:
+          type: object
+          properties:
+            lumens:
+              type: integer
+              minimum: 0
+              description: Light intensity measured in lumens.
+            sentAt:
+              $ref: "#/components/schemas/sentAt"
+        sentAt:
+          type: string
+          format: date-time
+          description: Date and time when the message was sent.
+
+      securitySchemes:
+        saslScram:
+          type: scramSha256
+          description: Provide your username and password for SASL/SCRAM authentication
+
+      parameters:
+        streetlightId:
+          description: The ID of the streetlight.
+          schema:
+            type: string
+
+If you want to look at it in a more readable way, you can copy and paste the definition to the [AsyncAPI Studio](https://studio.asyncapi.com/) or the [AsyncAPI playground](https://playground.asyncapi.io)(*Remark*: AsyncAPI Playground will soon be archived and replaced by AsyncAPI Studio). More examples are available on [Github](https://github.com/asyncapi/spec/tree/master/examples).
+
+I do not want to dive too deep into the exact details of the specification because the [documentation](https://www.asyncapi.com/docs/specifications/v2.2.0) is already helpful enough for this. But I want to highlight some parts of the specification that could be the icing on the cake when considering it.
+
+Besides some general information about the API, like the title and the description, the "Info" object can contain a ["Contact" object](https://www.asyncapi.com/docs/specifications/v2.2.0#contactObject). Especially when thinking about multiple groups working together, the responsible team and how to reach them can be linked at this place. In the example from above, this would look like:
```

Comment:
```suggestion
Besides some general information about the API, like the title and the description, the "Info" object can contain a ["Contact" object](https://www.asyncapi.com/docs/specifications/v2.2.0#contactObject). Especially when thinking about multiple groups working together, the responsible team and how to reach them can be linked at this place. In the example from above, this would look like this:
```